SUMMARY.md

```diff
@@ -0,0 +1,28 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can use a specific class or command to establish a connection between PHP and PostgreSQL.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP Data Objects (`PDO`) classes and how to execute queries using `PDO::query()`.
+
+* [Prepared Statement](content/PreparedStatement/PreparedStatement.q.md) ** - minutes**
+    > In this lesson, you will learn the basics of using `PDO::prepare()` with its parameters in PHP.
+
+* [Prepared Statement with execute()](content/UsingExecute/UsingExecute.q.md) ** - minutes**
+    > Discover how you can execute prepared statements using `PDOStatement::execute()` with binding methods such as `bindParam()` and `bindValue()`.
+
+* [Prepared Statement with fetch()](content/UsingFetch/UsingFetch.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetch()` to fetch data and return a specific row of data from a result set.
+
+* [Prepared Statement with fetchColumn()](content/UsingFetchColumn/UsingFetchColumn.q.md) ** - minutes**
+    > Learn how you can use `PDOStatement::fetchColumn()` to return one column field from the next row of a result set.
+
+* [Prepared Statement with fetchAll()](content/UsingFetchAll/UsingFetchAll.q.md) ** -minutes**
+    > Discover how you can use `PDOStatement::fetchAll()` to return an array that holds all the rows in the result set.
```

Comment:
Please replace `the` with `a` in this phrase `in the result set` to become __`in a result set`__. Thanks.